src/bin/main.rs

```diff
@@ -1,6 +1,6 @@
 extern crate config;
 
-use config::*;
+use config::Config;
```

Comment:
Much like the whitespace changes in `Cargo.toml`, this isn't strictly necessary, but I don't see much wrong with it either.